test/browser_tests/page_objects/page_careers-application-process.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
```

Comment:
Works for me. I just searched the project and turns out we didn't even use underscore anywhere. I should have done that first but I swore we had used it at least once.